roop/enhancer.py

```diff
@@ -149,7 +151,8 @@ def process_frames(source_path: str, frame_paths: list[str], progress=None) -> N
         try:
             frame = cv2.imread(frame_path)
             result = process_faces(source_face, frame)
-            cv2.imwrite(frame_path, result)
+            cv2.imwrite(state.get_frame_processed_name(frame_path), result)
```

Comment:
It would make send to add a variable that points to state.get_frame_processed_name(temp_frame_path) so the reader can easier understand what it contains

Reply:
done